gui/wxpython/core/treemodel.py

```diff
@@ -294,15 +294,12 @@ def __init__(self, label=None, data=None):
     def label(self):
         return self._label
 
-    def match(self, key, value, case_sensitive=False):
+    def match(self, key, value, case_sensitive=False) -> bool:
```

Comment:
Do we need `from __future__ import annotations` here?

Reply:
No, the future is used when we want to use `|` instead of `Optional[]` (that needs to be imported too), plus more other syntax that was supposed to be mandatory in 3.10, but still postponed. (That future exists since 3.7).

https://docs.python.org/3/library/__future__.html#module-contents

It also uses the new behavior of considering the types as strings without evaluating for better performance, instead of evaluating the types. Type checking doesn't isn't enforced at runtime anyways, so post-poneing makes sense.

In this file, bool doesn't need extra workarounds to work correctly.

Type annotations isn't new, it was there in Python 3.5, but supported syntax evolved to be more useful